roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -0,0 +1,23 @@
+# Kubernetes doesn't use containerd restart manager.
+disabled_plugins = ["restart"]
+
+[debug]
+  level = "info"
+
+[plugins.linux]
+  shim = "/usr/bin/containerd-shim"
+  runtime = "/usr/sbin/runc"
+
+[plugins.cri]
+  stream_server_address = "127.0.0.1"
+  max_container_log_line_size = 102400
+[plugins.cri.cni]
+  bin_dir = "/opt/cni/bin"
+  conf_dir = "/etc/cni/net.d"
+  conf_template = ""
+# [plugins.cri.registry.mirrors."docker.io"]
+#  endpoint = ["https://mirror.gcr.io","https://registry-1.docker.io"]
```

Comment:
are this 2 line needed? if so why they are comment out ?

Reply:
Now removed.